### DISCUSSION TITLE
Further zoom out (as alternative to minimap?)

### DISCUSSION THREAD

**Discussion by SRS-CHI** [voxeltycoon/issues#962] [General]
I see that Minimap has been on the roadmap since 2019.  It may also suffice to allow a much further zoom out.  After several regions are annexed, the inability to zoom out to see most of your area makes it difficult to monitor the big picture.  Either the ability to zoom out much further or a minimap would help with this problem.

**Comment by gralewski** [comment]
Yes, I second that! Please unlock zoom out.
